src/V3AstNodes.cpp

```diff
@@ -1283,6 +1283,28 @@ string AstBasicDType::prettyDTypeName() const {
 void AstNodeMath::dump(std::ostream& str) const { this->AstNode::dump(str); }
 void AstNodeUniop::dump(std::ostream& str) const { this->AstNodeMath::dump(str); }
 
+std::string AstStrength::name() const {
+    string strengthString;
+    switch (this->m_level) {
+    case HIGHZ: strengthString = "highz"; break;
+    case SMALL: strengthString = "small"; break;
```

Comment:
```suggestion
    char valChar = val ? '1' : '0';
    switch (m_level) {
    case HIGHZ: return "highz" + valChar;
    case SMALL: return "small" + valChar;
```
and so on.

Though the Verilator maintainers may want this to look like other enums, e.g. `VEdgeType`. See the `ascii` method there.

Reply:
Done. I also get rid of `val` field.